build-release-package.sh

```diff
@@ -48,6 +48,8 @@ rm -rf $LAMBDA_BUILD_DIR
 mkdir $PACKAGE_BUILD_DIR
 
 cp README.md dynatrace-aws-logs.sh $LAMBDA_ZIP_NAME dynatrace-aws-log-forwarder-template.yaml $PACKAGE_BUILD_DIR
-zip $PACKAGE_ZIP_NAME $PACKAGE_BUILD_DIR/*
+cd $PACKAGE_BUILD_DIR
+zip ../$PACKAGE_ZIP_NAME *
+cd ..
```

Comment:
before the structure was:
dynatrace-aws-log-forwarder.zip/dynatrace-aws-log-forwarder/<files>
now it is:
dynatrace-aws-log-forwarder.zip/<files>

so that the unzip doesn't produce unzip dynatrace-aws-log-forwarder/<files> from the zip, but only <files> directly to the current dir

I'm more fine with 4 files spamming user's dir than with secretly spamming the user with multiple levels of dirs lower and lower.

Reply:
This bears some signs of a tarbomb but I guess four files is not that bad really, especially since they should have names different than other files in the user's directory and so shouldn't overwrite anything. Fine by me